src/equations/2d/compressible_euler.jl

```diff
@@ -383,6 +384,8 @@ function boundary_conditions_sedov_self_gravity(u_inner, orientation, direction,
 end
 
 # Apply source terms
+# TODO: Taal remove methods with the signature below?
+#       Or keep them as an option for possiby increased performance?
```

Comment:
I think this should become a separate issue and could be a good student project.

Reply:
Sounds good to me. Can you open an issue for that, please?

Reply:
Tracked in #208